src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -111,7 +127,7 @@ void PerformanceImplicitGemmForwardV4R4Xdlops::EuristicInit(const ConvolutionCon
     auto get_euristic_config = [&](auto is_valid_func) {
         if(ctx.IsFp32())
         {
-            tmp = {256, 256, 8, 128, 128, 4, false, true};
+            tmp = {256, 256, 8, 128, 128, 4, false, true, 1};
```

Comment:
hi artem, 
I see there's some slice difference between fp32, fp16, bfp16:
1. initial value of tmp
2. value region of GemmKPack
2. value region of GemmBThreadDataPerRead_GemmN as well.
I don't know why, maybe for special consideration, @asroy 
since there're so many difference here, I would prefer to keep code as before.
3.

Reply:
> I don't know why, 

But you should know because this is your PR. Please find out. When you figure it out, I will have no objection to leave the code as it is now (if you will want this).

Reply:
hi Artem,
thanks for your comments, I'll sync with chao to see if there's special consideration, if no, I would prefer to leave it here.

Reply:
@atamazov FP32, FP16, BFp16 are not using exactly the same tuning parameter heuristic, merging them together would still require logic divergence. I prefer keeping them in separate blocks instead of mixing them.

Reply:
Ok

Reply:
@asroy 
> FP32, FP16, BFp16 are not using exactly the same tuning parameter heuristic

Note that it would be good for @jane-zxy to know the reasons of the above.